gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/io/gryo/GryoVersion.java

```diff
@@ -321,12 +320,10 @@ public static List<TypeRegistration<?>> initV3Registrations() {
             add(GryoTypeReg.of(ProjectedTraverser.class, 168));
             add(GryoTypeReg.of(DefaultRemoteTraverser.class, 123, new GryoSerializersV3.DefaultRemoteTraverserSerializer()));
 
-            add(GryoTypeReg.of(GremlinLang.class, 122, new GryoSerializersV3.BytecodeSerializer()));
```

Comment:
rather than remove these lines wholly, please leave a comment in each space like:
```
// 122 was Bytecode until it was removed for 4.x
// 126 was Bytecode.Binding until it was removed 4.x
```